sisyphus/tools.py

```diff
@@ -82,6 +83,8 @@ def extract_paths(args: Any) -> Set:
     elif isinstance(args, (list, tuple, set)):
         for a in args:
             out = out.union(extract_paths(a))
+    elif isinstance(args, enum.Enum):
+        return out
```

Comment:
`Block` and `Enum` have the same operation. Maybe merge?

Reply:
would be possible yes, I don't know what is better... saving a line or having potentially more checks...

Reply:
I'd group by behavior and the behavior for Block and Enum is the same here and I don't see that changing anytime in the future.